### ISSUE TITLE
Issues setting up Bonobo on Win 10

### ISSUE THREAD

**Issue by J-Farmer** [jakubgarfield/Bonobo-Git-Server#697]
Hello All, 

I have followed the installation instructions listed at [](https://bonobogitserver.com/install/) precisely, and I still am getting the HTTP Error 403.14 - Forbidden when trying to connect. I have also registered my .NET Framework with my IIS. I am using the Default Application Pool in IIS, which is a 4.0 .NET framework and the pipeline is Integrated. I'm at a loss here, and any help would be appreciated. Edit: I'm trying to set up Bonobo Version 6.2.1

**Comment by willdean** [comment]
@J-Farmer It's really difficult for us to help with IIS problems, because there are so many possible causes.  However, there may be something useful here: #456, or there maybe something useful in one of these:

https://stackoverflow.com/questions/1741439/asp-mvc-in-iis-7-results-in-http-error-403-14-forbidden
https://stackoverflow.com/questions/15323190/http-error-403-14-forbidden-iis-error-for-asp-net-mvc-4-application

I would suggest you start by installing the latest .NET Framework (or at least 4.6.2 if you don't want to do 4.7), and then create a new app-pool from scratch and assign Bonobo to that new - pool #456 may help there.

**Comment by J-Farmer** [comment]
Issue Resolved: I went into Turn Windows Features on/off and activated ASP.NET under IIS->World Wide Web Services -> Application Development Features. I can now view the login screen for Bonobo.

**Comment by Reederoo** [comment]
Thank you @J-Farmer . Your last comment fixed my issue as well and saved me much time and frustration. This really should be part of the installation instructions for Windows 10 installations, shouldn't it?

**Comment by rryi** [comment]
fixed it for me, too.

For my windows 10 professional "server" with no IIS use before, I had to perform the following steps in addition to the https://bonobogitserver.com/install/  walkthru

(0) install IIS features: management console plus www services. I noticed these two checked boxes do NOT include all possible components which are shown if I expand the subtrees. I just left it as is for now.

Then: installation steps in https://bonobogitserver.com/install/ 
Open http://localhost/Bonobo.Git.Server in browser gives http Error 500.19 error.

(1) fix error according to FAQ tip:
%windir%\system32\inetsrv\appcmd unlock config -section:system.webServer/handlers
%windir%\system32\inetsrv\appcmd unlock config -section:system.webServer/modules

Open http://localhost/Bonobo.Git.Server in browser gives http Error 403.14

(2) activated ASP.NET under IIS->World Wide Web Services -> Application Development Features.
Open http://localhost/Bonobo.Git.Server in browser shows now Bonobo sign in page.

**Comment by SChalice** [comment]
> This really should be part of the installation instructions for Windows 10 installations, shouldn't it?

Yes

**Comment by JulioImolesi** [comment]
@rryi  
**"(0) install IIS features: management console plus www services. I noticed these two checked boxes do NOT include all possible components which are shown if I expand the subtrees. I just left it as is for now."**

I already was looking for another git server, I've stuck in HTTP Error 403.14 then I saw this phrase and everything made sense. At installation I just check the main checkboxes. When I back to IIS features a lot of checkboxes was unchecked, so I check them all, restarted IIS and everything works fine!
Thankz again @rryi !!!
